### ISSUE TITLE
Why transpose(1, 3) before BatchNorm?

### ISSUE THREAD

**Issue by ado-ml** [qiuqiangkong/audioset_tagging_cnn#39]
May I ask why do transpose(1, 3) before BN? Is it intended to do batch normalization for each frequency bin, what is the advantage for this? Thanks. 

x = x.transpose(1, 3)
x = self.bn0(x)
x = x.transpose(1, 3)

**Comment by qiuqiangkong** [comment]
That is equivalent to normalize each frequency bin. For example, if there are 64 mel bins, then there are 64 mean values and 64 standard values for normalization.
